_posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "2018-12-02-Reading: 7 reasons why smart, hardworking people don't become successful"
+date:   2018-12-02 19:31:07 +0800
+categories: readingnote
+---
+
+[Article Link](2018-12-02-Reading: 7 reasons why smart, hardworking people don't become successful)
```

Comment:
Similarly please include a detailed reference to the article you read, I am sure there will be people who are interested in reading that too.

Reply:
t's a TED I watched and I just attached the video link.